src/backend/dist.js

```diff
@@ -223,62 +223,112 @@ module.exports = {
     unzipper.on('extract', function(log) {
       logger.addLog('Info', 'zip successfully extracted', socket);
 
-      fs.readdir(appPath + '/zip' , function(err, list){
-        if(err) {
-          logger.addLog('Error', 'Error while reading directory', socket, err);
-          return done(err);
+        async.series([
+
+          // Resizing sponsors images to 150X80
+          function(callback) {
+            fs.readdir( appPath + '/zip/images/sponsors', function(err, list){
+              if(err) {
+                logger.addLog('Info', 'No sponsors images found', socket, err);
+                return 0;
+              }
+              list.forEach( function( image ){
+                sharp( appPath + '/zip/images/sponsors/' + image)
+                  .resize(150, 80)
+                  .background({r: 255, g: 255, b: 255, a: 0})
```

Comment:
The size conversion of images works embed method, i.e, a frame of 150X80 is created then the image is embedded into it. After that, all left whitespace in the frame is filled with transparent background and in case if it is not png then filled with white color.